tests/unit_tests/test_public_api.py

```diff
@@ -212,6 +212,31 @@ def test_artifact_download_logger():
             termlog.assert_not_called()
 
 
+def test_update_aliases_on_artifact(user, relay_server, wandb_init):
+    project = "test"
+    run = wandb_init(entity=user, project=project)
+    artifact = wandb.Artifact("test-artifact", "test-type")
+    with open("boom.txt", "w") as f:
+        f.write("testing")
+    artifact.add_file("boom.txt", "test-name")
+    run.log_artifact(artifact, aliases=["best"])
+    artifact.wait()
+    run.finish()
+
+    artifact = Api().artifact(
+        name=f"{user}/{project}/test-artifact:v0", type="test-type"
+    )
+    artifact.update_aliases(add=["staging"], remove=["best"])
```

Comment:
When you remove an alias, does it go to the previous assigned artifact? or does that alias completely go away?

Reply:
The alias goes away until the user re-adds it to an artifact in the collection. This is the current expected behavior.

Reply:
ok - as long as that is your intention!